definition/latitude_longitude_properties.md

```diff
@@ -0,0 +1,65 @@
+## Various latitude/longitude properties
+
+### geom
+
+Find out more about the `geom` property [here](https://github.com/whosonfirst/whosonfirst-properties/tree/master/properties/geom).
+
+The `geom:latitude` and `geom:longitude` represent a geometric centroid for a record.
+
+These values are machine-generated math centroids for a record's geometry.
+
+### intersection
+
+Find out more about the `intersection` property [here](https://github.com/whosonfirst/whosonfirst-properties/tree/master/properties/intersection).
+
+The `intersection:latitude` and `intersection:longitude` are used in [intersection records](https://whosonfirst.mapzen.com/spelunker/placetypes/intersection/) to indicate the location of an intersection. 
+
+The values are latitude and longitude values from the source data used to generate the Voronoi intersection polygons.
+
+### lbl
+
+Find out more about the `lbl` property [here](https://github.com/whosonfirst/whosonfirst-properties/tree/master/properties/lbl).
+
+The `lbl:latitude` and `lbl:longitude` represent an optional, curated location for a given record where the label or search result pin should be placed.
```

Comment:
The `lbl:latitude` and `lbl:longitude` represent an optional, curated location for a given record where the label or search result pin should be placed. When not curated it's often machine generated by MapShaper. When it's curated this property must be added to the controlled list to preserve the value during an auto-update.

Something something.